crates/burn-wgpu/src/runtime.rs

```diff
@@ -51,7 +46,8 @@ impl<G: GraphicsApi> Runtime for WgpuRuntime<G> {
 
     fn client(device: &Self::Device) -> ComputeClient<Self::Server, Self::Channel> {
         RUNTIME.client(device, move || {
-            let (adapter, device_wgpu, queue) = pollster::block_on(create_wgpu_setup::<G>(device));
+            let (adapter, device_wgpu, queue) =
+                pollster::block_on(create_wgpu_setup::<AutoGraphicsApi>(device));
```

Comment:
So if I understand it well, this is where the default runtime is actually created for a device type. It uses the `AutoGraphicsApi` to select the most appropriate adaptor for the device. If you want to specify manually the graphics API for a device type, you have to use `init_sync` or `init_async`!

I don't think this is a bad thing, now the `WgpuBackend` only has typical backend generics, and runtime options can be changed using the init functions! Before merging I would add a bit more documentation on the `WgpuRuntime` and `WgpuBackend` pointing to the right direction to specify backend and runtime options.

Reply:
Yep that's right! Seemed coherent to me to have one "advanced initialize" place, glad you agree :) 

Have added some docs now, and a random code cleanup, lmk how this looks!